### ISSUE TITLE
Could not load file or assembly on linux

### ISSUE THREAD

**Issue by shynome** [egramtel/dotnet-bundle#2]
I have run the below command on my linux os:
```sh
dotnet restore -r osx-x64
dotnet msbuild -t:BundleApp -p:RuntimeIdentifier=osx-x64
```
then throw some error
```log
~/.nuget/packages/dotnet.bundle/0.9.12/build/DotNet.Bundle.targets(42,5): error MSB4062: The "DotNet.Bundle.BundleAppTask" task could not be loaded from the assembly ~/.nuget/packages/dotnet.bundle/0.9.12/build/../tasks/netstandard2.0/DotNet.Bundle.dll. Could not load file or assembly '~/.nuget/packages/dotnet.bundle/0.9.12/tasks/netstandard2.0/DotNet.Bundle.dll'. The system cannot find the file specified. [~/myproject/myproject.fsproj]
~/.nuget/packages/dotnet.bundle/0.9.12/build/DotNet.Bundle.targets(42,5): error MSB4062:  Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [~/myproject/myproject.fsproj]
```

I try to find why and I have found, because the dir `~/.nuget/packages/dotnet.bundle/0.9.12/tasks/netstandard2.0/` have the file `Dotnet.Bundle.dll` not `DotNet.Bundle.dll` , because file names in Linux are case sensitive but windows not so the error is only throw on linux , I have rename `Dotnet.Bundle.dll` to `DotNet.Bundle.dll` and run again everything is working

**Comment by x2bool** [comment]
Hi! Thanks! Indeed there are naming issues. I'll fix this.

**Comment by x2bool** [comment]
Just released 0.9.13 with a fix. Thanks!
